core/vm/logger.go

```diff
@@ -66,19 +65,16 @@ type LogConfig struct {
 
 // StructLog is emitted to the EVM each cycle and lists information about the current internal state
 // prior to the execution of the statement.
+// [kroma] : An error occurred when going:generate, so use the old code to restore it. StructLog.Memory, StructLog.ReturnData
```

Comment:
I couldn't do a proper upstream because go:generate was throwing an error because of the changes in the scroll, so I reverted back to the op-geth baseline and removed the scroll marks.

Reply:
I'll make the capitalization change later, all at once.